workflow/pipeline/events.py

```diff
@@ -3,8 +3,11 @@
 from workflow import db_prefix
 from workflow.pipeline import lab, session
 from workflow.utils.paths import (get_ephys_root_data_dir,
-                                  get_processed_root_data_dir,
-                                  get_session_directory)
+                                  get_ephys_processed_root_data_dir,
+                                  get_imaging_root_data_dir,
+                                  get_imaging_processed_root_data_dir,
+                                  get_imaging_session_directory,
+                                  get_ephys_session_directory)
```

Comment:
Do we need all of this to activate `event` and `trial` schemas?

Reply:
I don't think it is all needed, was a bit confused as to where these functions are being used within the activation, so I included all of them for now.

Reply:
let's not include any for now, let's figure out which ones are used where and just include those.